assets/js/src/newsletters/send/congratulate/success.jsx

```diff
@@ -16,10 +16,28 @@ function renderHeader(newsletter) {
 }
 
 function Success(props) {
+  const showSuccessDeliveryPoll = (
+    props.newsletter.type === 'standard' &&
+    props.newsletter.status !== 'scheduled' &&
+    MailPoet.Poll.successDelivery.canShow('campaign')
+  );
+  if (showSuccessDeliveryPoll) {
+    MailPoet.Poll.successDelivery.initTypeformScript();
+    MailPoet.Poll.successDelivery.setPollShown('campaign');
```

Comment:
When the component is re-rendered this causes that the poll is hidden. We can set this on close button or maybe we can just rely on congratulations page display logic (which is the same) and remove additional condition for the poll.